### DISCUSSION TITLE
What is suspend?

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#8149] [Q&A]
What is suspend? Tried it on the tap and hold gesture and nothing happened. 

I'm using an android tablet Samsung galaxy Tab A (T-295) with the latest koreader version 8.2021 

![IMG_20210829_212537_451](https://user-images.githubusercontent.com/70109540/131261352-736adf19-2e98-4c02-a154-81a835583bb7.jpg)

**Comment by NiLuJe** [comment]
Essentially, sleep mode. Probably doesn't make much sense on android.

**Reply by Frenzie** [reply]
Yes, it showing is probably a bug.

Cf. for the main menu:

```lua
if Device:canSuspend() then
    common_info.sleep = {
        text = _("Sleep"),
        callback = function()
            UIManager:suspend()
        end,
    }
end
```

**Reply by Zeyadas** [reply]
Now I get it.. Thanks.
